app/code/Dev/ProductComments/view/frontend/templates/product/view/comments.phtml

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * @var $block View
+ */
+use Dev\ProductComments\Block\View;
+$currentProduct = $block->getCurrentProduct();
+$productId =  $currentProduct->getId();
+$commentsAvailable = $currentProduct->getData('ProductComment');
+$comments = $block->getCommentCollection($productId);
+$productName = $block->getProductName($productId);
+if (count($comments) > 0): ?>
+<div class="block review-list">
+    <div class="block-title">
+        <strong>Product Comments</strong>
+    </div>
+    <?php foreach ($comments as $item): ?>
+    <div class="block-content">
+        <ol class="items review-items">
+            <li class="item review-item" itemscope="" itemprop="review" itemtype="http://schema.org/Review">
+                <div class="review-content" itemprop="description"><?= $item->getData("body") ?></div>
+                <div class="review-details">
+                    <p class="review-author"></p>
+                    <p><b><?= $item->getData('comment')?></b></p>
+                    <span class="review-details-label">Commented by</span>
+                    <span class="review-details-value"
+                          itemprop="author"><?= $item->getData("name") ?></span>
+                    <br>
+                    <p class="review-date">
+                        <span class="review-details-label">Posted on</span>
+                        <time class="review-details-value"
+                              itemprop="datePublished"><?= date('m/j/Y', strtotime($item->getData("date"))) ?></time>
```

Comment:
Use core magento for date output. Magento\Framework\Stdlib\DateTime\TimezoneInterface